GPU/Common/TextureCacheCommon.cpp

```diff
@@ -2058,11 +2057,25 @@ bool TextureCacheCommon::PrepareBuildTexture(BuildTexturePlan &plan, TexCacheEnt
 	plan.scaleFactor = standardScaleFactor_;
 
 	// Rachet down scale factor in low-memory mode.
-	if (lowMemoryMode_) {
+	// TODO: I think really we should just turn it off?
```

Comment:
Maybe.  It also does purge textures - the thinking was that some people set it to 5x, run out of VRAM, but could handle a lower value.

-[Unknown]

Reply:
True, might be reasonable.